core-library/src/main/java/org/silverpeas/core/admin/service/Administration.java

```diff
@@ -1107,7 +1111,10 @@ String identify(String sKey, String sSessionId, boolean isAppInMaintenance, bool
    * @param inAllSpaces
    * @return Array of componentsIds
    * @author dlesimple
+   * @deprecated please use {@link #getAllComponentIds(String)} or
+   * {@link #getAllComponentIdsRecur(String)}
    */
+  @Deprecated
   String[] getAllComponentIdsRecur(String sSpaceId, String sUserId, String componentNameRoot,
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1133)